app-timer-run/src/main/java/xyz/aprildown/timer/app/timer/run/screen/ScreenActivity.kt

```diff
@@ -70,6 +73,17 @@ class ScreenActivity : BaseActivity() {
             }
         }
 
+        intent?.getStringExtra(EXTRA_IMAGE_PATH)?.let { imagePath ->
+            if (imagePath.isNotEmpty()) {
+                val image = File(imagePath)
+                if (image.extension == "gif")
+                    Glide.with(this).asGif().load(Uri.parse(imagePath)).into(binding.image)
+                else
+                    Glide.with(this).asBitmap().load(Uri.parse(imagePath)).into(binding.image)
+//                binding.image.setImageURI(Uri.parse(imagePath))
+            }
+        }
+
```

Comment:
Glide can take care of everything for you, like this:
```Glide.with(this).load(imagePath.toUri()).into(binding.image)```

Reply:
Good to know. Thanks!